src/auth.ts

```diff
@@ -4,18 +4,29 @@ import { Constants, getResourceIdFromPath, HTTPMethod, ResourceType } from "./co
 import { IHeaders } from "./queryExecutionContext";
 
 /** @hidden */
-export interface IRequestInfo {
-  [index: string]: any;
+export interface RequestInfo {
```

Comment:
If you're not using Request info in the signature of the method any more you don't need this class, but see my message below about whether we want to do that.

Reply:
Added this interface back